cms/envs/bok_choy.py

```diff
@@ -93,6 +93,8 @@
 FEATURES['ENABLE_MOBILE_REST_API'] = True  # Enable video bumper in Studio
 FEATURES['ENABLE_VIDEO_BUMPER'] = True  # Enable video bumper in Studio settings
 
+FEATURES['ENABLE_ENROLLMENT_TRACK_USER_PARTITION'] = True
```

Comment:
By the way, in the future, consider using a waffle switch in these cases.  This way, you wouldn't need to be blocked on a release in order to enable/disable the feature.  I've been using it for several features now, and created helper waffle utility classes: https://github.com/edx/edx-platform/blob/master/openedx/core/djangolib/waffle_utils.py